scripts/pylint_extensions.py

```diff
@@ -69,9 +70,6 @@ def visit_call(self, node):
 
         # Build the set of keyword arguments and count the positional arguments.
         call_site = astroid.arguments.CallSite.from_call(node)
-        if call_site.has_invalid_arguments() or (
```

Comment:
This line seems unreachable. Args and Kwargs are derived from _unpacked_args or _unpacked_kwargs so how can they not be equal? I looked at the source on pylint -- its mentioned that even they cannot figure it out. See:
https://github.com/PyCQA/pylint/blob/1ce0bbb1bd0deb6cd53a6d6c80f68193ad1817f6/pylint/checkers/typecheck.py#L1110